modules/data-entry/src/main/frontend/src/questionnaireEditor/AnswerOptions.jsx

```diff
@@ -323,6 +387,49 @@ let AnswerOptions = (props) => {
         multiline
         />
       { generateSpecialOptions(1) }
+      <Popover
+        disableBackdropClick
+        disableEscapeKeyDown
+        open={Boolean(descriptionAnchorEl)}
+        anchorEl={descriptionAnchorEl}
+        onClose={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'center',
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'center',
+        }}
+        className={classes.descriptionPopover}
+      >
+        <Typography variant="h6" className={classes.descriptionPopoverTitle} >
+          {"Description for " + descriptionLabel}
```

Comment:
The label itself should appear in quotation marks on the UI. This should do it (not tested):
```suggestion
          {`Description for "${descriptionLabel}"`}
```